rust/src/tftp/tftp.rs

```diff
@@ -31,11 +31,28 @@ pub struct TFTPTransaction {
     pub opcode : u8,
     pub filename : String,
     pub mode : String,
-    pub logged : LoggerFlags
+    pub logged : LoggerFlags,
+    id: u64,
 }
 
 pub struct TFTPState {
-    pub transactions : Vec<TFTPTransaction>
+    pub transactions : Vec<TFTPTransaction>,
+    /// tx counter for assigning incrementing id's to tx's
```

Comment:
nit: use 2 `/` characters

Reply:
In Rust ```///``` has a special meaning for documenting, so it may be correct here.